.github/workflows/build.yml

```diff
@@ -2,25 +2,26 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, v5.0.0-development ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, v5.0.0-development ]
 
 jobs:
   build:
 
     runs-on: ubuntu-latest
 
     strategy:
+      fail-fast: false
       matrix:
-          node-version: [10.x, 12.x, 14.x, 16.x]
+          node-version: [14.x, 16.x, 18.x]
```

Comment:
Just a quick question here. According to the README.md, should this node-version also include 20?

Reply:
My mistake. README is wrong—Node 20 hasn't been released yet.

Reply:
> My mistake. README is wrong—Node 20 hasn't been released yet.

Other than that, looks good to me.